docs/ops-recipe/harborUnhealthy.md

```diff
@@ -0,0 +1,93 @@
+---
+
+title: "Harbor Unhealthy"
```

Comment:
This is almost the same as the 'failing' one. Should they be merged? Having copies doesn't help anyone and makes changes more complex